http_prompt/printer.py

```diff
@@ -0,0 +1,20 @@
+import click
+
+
+class Printer():
```

Comment:
if it's not necessary to use the `click` library due to compatibility reasons, we can write directly to `stdout`.... I guess it's better this way...